backend/t4cclient/core/authentication/provider/oauth/__main__.py

```diff
@@ -1,11 +1,14 @@
-from t4cclient.config import config
+# Standard library:
+import typing as t
 
+# local:
 from .keystore import KeyStore
+from t4cclient.config import config
 
 cfg = config["authentication"]["oauth"]
 
 
-def get_jwk_cfg(token: str) -> dict[str, any]:
+def get_jwk_cfg(token: str) -> t.Dict[str, any]:
```

Comment:
Type hinting in standard collection was introduced with PEP 585 in Python 3.9. 
We should rather set the minimum Python version to 3.9.

Reply:
But when I check in Python 3.8, the typing module contains a Dict generic, and also since dict is a type, it's not (and has never been) subscriptable, that means that if we want to use dict instead of Dict, we cannot specify the type of the keys.

Reply:
We can either use
```python
def get_jwk_cfg(token: str) -> dict:
```
or 
```python
def get_jwk_cfg(token: str) -> t.Dict[str, any]:
```
but no
```python
def get_jwk_cfg(token:str) -> dict[str, any]:
```